packages/react/src/components/Card/CardMedia/Video/index.js

```diff
@@ -25,76 +25,69 @@ const Video = styled.video`
   `};
 `
 
-class CardVideo extends Component {
-  constructor (props) {
-    super(props)
-    this.state = {
-      playing: props.autoPlay,
-      progress: 0
-    }
-  }
+function CardVideo (props) {
```

Comment:
I think we could destructure `props` here directly, instead of lines 49-60 (and avoid writing `props.X` multiple times).